test/gtest/common/test.cc

```diff
@@ -32,10 +32,11 @@ test_base::test_base() :
     push_config();
 }
 
-static void dofork_cb(ucs_mem_info_t* info, void *ctx) {
-    if (info->flags & UCS_MEM_FLAG_DONTCOPY) {
-        EXPECT_EQ(0, madvise((void *)info->start, info->end - info->start,
-                             MADV_DOFORK));
+static void dofork_cb(ucs_sys_vma_info_t *info, void *ctx) {
+    if (info->flags & UCS_SYS_VMA_FLAG_DONTCOPY) {
+        EXPECT_EQ(0, madvise((void*)info->start, info->end - info->start,
+                    MADV_DOFORK)) << "errno: " << errno << " "
```

Comment:
maybe to simplify:
`int ret = madvise((void*)info->start, info->end - info->start, MADV_DOFORK);`